tests/transformers/test_transformers_model_export.py

```diff
@@ -3924,3 +3627,92 @@ def test_text_generation_task_chat_serve(text_generation_pipeline):
         and output_dict["usage"]["completion_tokens"] < 10
     )
     assert output_dict["usage"]["prompt_tokens"] < 20
+
+
+def test_save_base_model_weight_false_pipeline(text_generation_pipeline, model_path):
+    MODEL_SHA = "38cc92ec43315abd5136313225e95acc5986876c"  # Hard coded sha for distilgpt2
+
+    with mock.patch("huggingface_hub.HfApi") as hf_api_mock:
+        hf_api_mock().model_info.return_value.sha = MODEL_SHA
```

Comment:
Do we need this `mock.patch`? What would happen if we remove it?

Reply:
oh wait this is the stale test and thought I removed it, probably mixed during rebasing...